tests/structuredheadings/formatcombo.js

```diff
@@ -0,0 +1,76 @@
+/* bender-tags: editor,unit */
+/* bender-ckeditor-plugins: wysiwygarea,structuredheadings,toolbar,basicstyles,dialog,richcombo */
+
+// Clean up all instances been created on the page.
+(function () {
+
+  bender.editor = {
+    allowedForTests: "h1; h2; p"
+  };
+
+  bender.test({
+    setUp: function () {
+      //Anything to be run before each test if needed
+    },
+
+    "if no headings in document, the new heading is autonumbered": function () {
+      var bot = this.editorBot;
+      var ed = this.editor;
+      bot.setHtmlWithSelection("<p>^foo</p>");
```

Comment:
I was initially very confused. It looked like we were converting all `p` tags to `h2` tags. But turns out that's not the case (since the cursor is in the `p` tag and we are enabling the plugin). To note however, I don't see any tests where a `p` tag stays a `p` tag. Do you think one is needed? Or are there existing tests that already cover this?